csv2rdf/csv2rdf_single_subject.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import validators
+import sys
+import json
+import os
+from rdflib import Graph, URIRef, Literal
+from rdflib.namespace import RDF
+from typing import Optional, List
+
+DIRNAME = os.path.dirname(__file__)
+mapping_filename = os.path.join(DIRNAME, 'mapping.json')
+dest_filename = os.path.join(DIRNAME, 'out_rdf.ttl')
```

Comment:
`mapping.json` does not correspond to any of the other files in this PR, which makes me wonder again whether these two wouldn't be better as command-line arguments.

Reply:
It's also a mistake in the rebasing process. I missed one commit.